src/bci_build/package/__init__.py

```diff
@@ -1108,6 +1108,7 @@ def description(self) -> str:
         - ``{pretty_name}``: the value of the pretty_name property
         - ``{based_on_container}``: the standard "based on the $distro Base Container Image" suffix that descriptions have
         - ``{podman_only}``: "This container is only supported with podman."
+        - ``{privileged_only}``: "This container is only supported in privileged mode."
```

Comment:
nit: why caps, quotes, and a dot in the end while some others above have not? there's an inconsistency of style in this section

Reply:
because the stuff in quotes is the literal string, while everything else is a description of what the string is.